docs/guide-end-of-form.rst

```diff
@@ -72,12 +73,12 @@ Auto send setting
 
 We generally recommend turning on :guilabel:`auto send` in :ref:`form management settings <form-management-settings>`. When :guilabel:`auto send` is on, Collect attempts to send filled forms as soon as they are finalized. The benefits of :guilabel:`auto send` are:
 
-* reduced risk of data collectors forgetting to submit data in a timely way.
-* automatically retry failed submissions. On poor or intermittent data connections, this can be very helpful.
+* reduced risk of data collectors forgetting to send data in a timely way.
+* more opportunities to retry sending. On poor or intermittent data connections, this can be very helpful.
```

Comment:
"Very helpful on poor or intermittent Internet connections" reads better to me. Data connections sounds weird for a data collection app.

Reply:
Pretty sure you wrote that one...